sympy/physics/mechanics/_geometry.py

```diff
@@ -404,6 +389,7 @@ def geodesic_length(self, point_1: Point, point_2: Point) -> ExprType:
         cylinder, connecting two points can be calculated using the formula:
 
         .. math::
+
             l = \acos{\mathbf{v}_1 \dot \mathbf{v}_2}
```

Comment:
This is the same equation as the sphere, are they the same?

Reply:
Good spot. This is a copy-paste error.

Reply:
Removed.